profiles/build.profile.js

```diff
@@ -22,7 +22,6 @@ var profile = {
                 'dojox/gfx/svgext',
                 'dojox/gfx/shape'
             ],
-            targetStylesheet: 'app/resources/App.css',
```

Comment:
Are you sure we don't need this? I think we do so the dojo build compresses our CSS.